src/gui/src/AboutDialog.cpp

```diff
@@ -49,6 +49,8 @@ AboutDialog::AboutDialog(QWidget* parent)
 	about += tr( "<b>Project page</b><br>");
 	about += "http://sourceforge.net/projects/hydrogen<br><br>";
 	about += tr( "<b>Mailing lists</b><br>");
+	about += "http://lists.sourceforge.net/lists/listinfo/hydrogen-users<br>";
+	about += "http://lists.sourceforge.net/lists/listinfo/hydrogen-announce<br>";
```

Comment:
The announce mailing list was listed there until last year, i have removed it. Those mailing lists are really dead. Btw, both are still listed on the homepage, so i can't see why that they are not advertised.